css/style.css

```diff
@@ -9,285 +15,2164 @@ body {
+  .total-look__arrow {
+    left: 0;
+    margin-left: 80%;
+  }
+}
+
+@media (max-width: 1360px) {
+  .letters-yellow {
+    left: 1050px;
+  }
+}
+
+@media (max-width: 1360px) {
+  .letters-black {
+    left: 1100px;
+  }
+}
+
+@media (max-width: 1360px) {
+  .name {
+    left: 930px;
+  }
+}
+
+@media (max-width: 1360px) {
+  .bag-title-bold {
+    font-size: 30px;
+    margin-left: 25px;
+  }
+}
+
+@media (max-width: 1360px) {
+  .bag-title {
+    font-size: 18px;
+    margin-left: 25px;
+  }
+}
+
+@media (max-width: 1360px) {
+  .total-look__text {
+    font-size: 18px;
+    left: 25px;
+  }
+}
+
+@media (max-width: 1360px) {
+  .total-look__button-number {
+    width: 20%;
+    left: 25px;
+  }
+}
+
+@media (max-width: 1360px) {
+  .total-look__button {
+    width: 55%;
+    left: 140px;
+    height: 47px;
+  }
+}
+
+@media (max-width: 1320px) {
+  .total-look__arrow {
+    width: 5%;
+    margin-left: 82%;
+  }
+}
+
+@media (max-width: 1200px) {
+  .letters-yellow {
+    left: 850px;
+  }
+}
+
+@media (max-width: 1200px) {
+  .letters-black {
+    left: 900px;
+  }
+}
+
+@media (max-width: 1200px) {
+  .name {
+    left: 730px;
+  }
+}
+
+@media (max-width: 1150px) {
+  .total-look__button {
+    left: 120px;
+  }
+}
+
+@media (max-width: 1150px) {
+  .total-look__text {
+    width: 80%;
+  }
+}
+
+@media (max-width: 1170px) {
+  .total-look__arrow {
+    margin-left: 84%;
+    width: 4%;
+  }
+}
+
+@media (max-width: 1170px) {
+  .things {
+    width: 57%;
+  }
+}
+
+@media (max-width: 1170px) {
+  .total-look__link {
+    font-size: 12px;
+  }
+}
+
+@media (max-width: 1020px) {
+  .total-look__arrow {
+    margin-left: 85%;
+    width: 4%;
+  }
+}
+
+@media (max-width: 980px) {
+  .total-look__card {
+    height: 280px;
+  }
+}
+
+@media (max-width: 980px) {
+  .bag-title-bold {
+    font-size: 22px;
+  }
+}
+
+@media (max-width: 980px) {
+  .bag-title {
+    font-size: 13px;
+    top: 70px;
+  }
+}
+
+@media (max-width: 980px) {
+  .total-look__text {
+    font-size: 13px;
+    top: 110px;
+  }
+}
+
+@media (max-width: 980px) {
+  .total-look__button {
+    width: 54%;
+    left: 110px;
+    font-size: 10px;
+    top: 205px;
+  }
+}
+
+@media (max-width: 980px) {
+  .total-look__button-number {
+    top: 205px;
+  }
+}
+
+@media (max-width: 980px) {
+  .total-look__link {
+    font-size: 11px;
+    top: 2490px;
+  }
+}
+
+@media (max-width: 980px) {
+  .total-look__arrow {
+    top: 2490px;
+  }
+}
+
+@media (max-width: 980px) {
+  .letters-yellow {
+    left: 690px;
+  }
+}
+
+@media (max-width: 980px) {
+  .letters-black {
+    left: 740px;
+  }
+}
+
+@media (max-width: 980px) {
+  .name {
+    left: 570px;
+  }
+}
+
+@media (max-width: 900px) {
+  .total-look__link {
+    font-size: 10px;
+    max-width: 250px;
+  }
+}
+
+@media (max-width: 800px) {
+  .name,
+  .letters-yellow,
+  .letters-black,
+  .total-look__button-number {
+    display: none;
+  }
+}
+
+@media (max-width: 800px) {
+  .total-look {
+    height: 1120px;
+  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5449357)